apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes/ast.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST do
```

Comment:
I'd add some conditions to matching expressions in this module. We can filter some false positives if we validate that in `{call, _, args} when call not in [:., :__aliases__, :"::", :{}, :|>]`

Reply:
Nice. Could you give some examples which break over the current code, so I can add them as test cases?

Reply:
One of your suggested cases needed this to work :)

Reply:
I don't have examples off the top of my head but we use this check in elixir_sense https://github.com/elixir-lsp/elixir_sense/blob/7fb31589e9dfdb91c6498e1169a13a0fe056d15b/lib/elixir_sense/core/metadata_builder.ex#L30